yoticlient.go

```diff
@@ -265,7 +258,6 @@ func handleSuccessfulResponse(responseContent string, key *rsa.PrivateKey) (user
 			log.Printf("Unable to get 'Formatted Address' from 'Structured Postal Address'. Error: %q", err)
 		} else if formattedAddress != "" {
 			if _, err = profile.StructuredPostalAddress(); err != nil {
-				errStrings = append(errStrings, err.Error())
```

Comment:
errStrings was null before this. And then we immediately return, so length is again 1